src/verifiers/documentStatus/documentStore/ethereumDocumentStoreStatus.ts

```diff
@@ -217,97 +218,147 @@ export const isRevokedOnDocumentStore = async ({
   }
 };
 
-const isWrappedV2Document = (document: any): document is WrappedDocument<v2.OpenAttestationDocument> => {
-  return document.data && document.data.issuers;
+const skip: VerifierType["skip"] = async () => {
+  return {
+    status: "SKIPPED",
+    type,
+    name,
+    reason: {
+      code: OpenAttestationEthereumDocumentStoreStatusCode.SKIPPED,
+      codeString:
+        OpenAttestationEthereumDocumentStoreStatusCode[OpenAttestationEthereumDocumentStoreStatusCode.SKIPPED],
+      message: `Document issuers doesn't have "documentStore" or "certificateStore" property or ${v3.Method.DocumentStore} method`,
+    },
+  };
 };
-export const openAttestationEthereumDocumentStoreStatus: Verifier<
-  WrappedDocument<v2.OpenAttestationDocument> | WrappedDocument<v3.OpenAttestationDocument>
-> = {
-  skip: () => {
-    return Promise.resolve({
-      status: "SKIPPED",
-      type,
+
+const test: VerifierType["test"] = (document) => {
```

Comment:
just to add that isWrappedV3Document is not correct as well (it doesn't do what you expect). We might need to discuss if we actually want to have such a helper in our lib :)

Reply:
"yes"